tools/testing/test_all.bat

```diff
@@ -185,7 +185,7 @@ do
   if [ $RunCodeCoverage == "true" ]; then
     find ./libraries -type f -name "*.cpp" -exec gcov {} \; &> /dev/null
     # Hide codecov output since it corrupts the log too much
-    codecov &> /dev/null
+    ./codecov
```

Comment:
Do we want to see all the codecov command line outputs? Not sure, since this will be a lot of output. It can be useful information though.

Reply:
yes, its probably better to dev/null it.